bin/create-k8s-config-maps-and-secrets

```diff
@@ -31,7 +31,7 @@ type: Opaque
 data:
   OAUTH_UID: $(echo -n "${OAUTH_UID}" | base64)
   OAUTH_PASSWORD: $(echo -n "${OAUTH_PASSWORD}" | base64)
-  EDL_USERNAME: $(echo -n "${EDL_USERNAME}" | base64)
-  EDL_PASSWORD: $(echo -n "${EDL_PASSWORD}" | base64)
+  EDL_USERNAME: $(echo -n "${EDL_USERNAME:-unset}" | base64)
```

Comment:
Do we have a note somewhere that this is being deprecated?  It'd be great if we could just delete it.